### PR TITLE
Client / Subscriber must be logged in to view  premium content

### PR DESCRIPTION
**Pull request by johanbounce**
# [PT Story](https://www.pivotaltracker.com/story/show/171811170)
```
As a Subscriber
In order to access the premium content
I would like to be able to log in and log out
```
## Changes proposed in this pull request:
* Added log in functionality with email and password
* Added log out functionality
* Macthed the login functionality to fit with the premium/free content  functionality

## What I have learned working on this feature:
* More knowledge on J-tockauth
* More redux experience

## New Packages
* J-tockauth

## Screenshots
Skärmavbild 2020-03-31 kl  13 52 29
Skärmavbild 2020-03-31 kl  13 52 06
Skärmavbild 2020-03-31 kl  13 51 48

### CODE REVIEW

File: src/components/SpecificArticle.jsx [hunk 1]
```diff
@@ -4,13 +4,16 @@ import { Box, Grommet, Button } from "grommet";
 import { grommet } from "grommet/themes";
 
 class SpecificArticle extends Component {
-  state = {
-    premiumUser: false
-  };
+  // state = {
+  //   premiumUser: false
+  // };
```
Comment (hmvitous) [inline src/components/SpecificArticle.jsx]: Commented out code here

**Review by kfrostare — APPROVED** [review]
LG /Kf

File: src/components/SpecificArticle.jsx (hunk 1 above)
Comment (johanbounce) [reply src/components/SpecificArticle.jsx]: Removed

**Comment by hmvitous** [comment]
Looks good to me

**Review by hmvitous — APPROVED** [review]

File: src/components/PaymentForm.jsx [hunk 2]
```diff
@@ -13,11 +13,16 @@ const PaymentForm = props => {
   const dispatch = useDispatch();
   const submitPayment = async event => {
     event.preventDefault();
+    let headers = JSON.parse(localStorage.getItem('J-tockAuth-Storage'));
     let stripeResponse = await props.stripe.createToken();
     let token = stripeResponse.token.id;
+    debugger
```
Comment (JaimeCrz) [inline src/components/PaymentForm.jsx]: deeddededededede DEBUGGER!

Comment (hmvitous) [inline src/components/PaymentForm.jsx]: :fire: :fire: :fire: :fire: :fire: :fire:

**Review by JaimeCrz — CHANGES_REQUESTED** [review]

File: src/components/PaymentForm.jsx (hunk 2 above)
Comment (JaimeCrz) [reply src/components/PaymentForm.jsx]: LGTM

**Review by JaimeCrz — APPROVED** [review]

**Pull request merged**